plugins/node/opentelemetry-instrumentation-bunyan/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@opentelemetry/instrumentation-bunyan",
+  "version": "0.15.0",
+  "description": "OpenTelemetry instrumentation for bunyan",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "precompile": "tsc --version",
+    "prepare": "npm run compile",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p ."
+  },
+  "keywords": [
+    "opentelemetry",
+    "logging",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "instrumentation"
```

Comment:
probably adding word `bunyan` might be good idea, wdyt ?

Reply:
Thanks, added it :slightly_smiling_face: